api/src/pcapi/sandboxes/scripts/creators/industrial/create_new_caledonia_offerers.py

```diff
@@ -0,0 +1,158 @@
+import datetime
+import decimal
+import logging
+
+import schwifty
+
+from pcapi.core.finance import factories as finance_factories
+from pcapi.core.finance import models as finance_models
+from pcapi.core.geography import factories as geography_factories
+from pcapi.core.offerers import factories as offerers_factories
+from pcapi.core.offerers import models as offerers_models
+from pcapi.core.offers import factories as offers_factories
+
+
+logger = logging.getLogger(__name__)
+
+
+def create_new_caledonia_offerers() -> None:
+    logger.info("create_new_caledonia_offerers")
+
+    _create_nc_active_offerer()
+    _create_nc_minimal_offerer()
+
+    logger.info("created New Caledonia offerers")
+
+
+def _create_nc_active_offerer() -> None:
+    address = geography_factories.AddressFactory(
+        street="11 Avenue James Cook",
+        postalCode="98800",
+        city="Nouméa",
+        latitude=-22.267957,
+        longitude=166.433846,
+        inseeCode="98818",
+        banId="98818_w65mkd_00011",
+        timezone="Pacific/Noumea",
+    )
+    offerer = offerers_factories.NotValidatedOffererFactory(
+        name="Structure calédonienne à Nouméa",
+        street=address.street,
+        postalCode=address.postalCode,
+        city=address.city,
+        siren="NC1230001",  # NC + RID7 (experimental)
+        allowedOnAdage=False,
+    )
+    venue = offerers_factories.VenueFactory(
+        managingOfferer=offerer,
+        pricing_point="self",
+        name="Lieu avec RIDET à Nouméa",
+        siret="NC1230001001XX",  # NC + RIDET + padding (experimental)
+        departementCode="988",
+        latitude=address.latitude,
+        longitude=address.longitude,
+        bookingEmail="venue.nc@example.net",
+        street=address.street,
+        postalCode=address.postalCode,
+        city=address.street,
+        banId=address.banId,
+        timezone=address.timezone,
+        venueTypeCode=offerers_models.VenueTypeCode.MUSEUM,
+        description="Lieu de test en Nouvelle-Calédonie",
+        contact__email="noumea.nc@venue.com",
+        contact__website="https://nc.venue.com/noumea",
```

Comment:
[micro-détail] 

```suggestion
        contact__email="noumea.nc@example.com",
        contact__website="https://nc.example.com/noumea",
```

idem plus bas, histoire de ne pas avoir des noms de domaine incontrôlés.